src/views/Notification/components/NotificationForm/index.jsx

```diff
@@ -57,7 +56,7 @@ const DistanceSearch = ({ store }) => {
  * @param {function(): void} props.onCancelButtonClick - Callback that gets called when the user
  *     clicks the cancel button
  * @param {function(): void} props.onNotifyButtonClick - Callback that receives the edited
- *     parcel when the user clicks the "Notify" button
+ *     parcel when the user clicks the "Notify" button. Triggers an alert popup on success.
```

Comment:
Maybe just change this to "Callback that gets called when the user clicks the "notify" button.
The callback no longer receives the edited parcel, and this module doesn't really need to know what happens up the component tree.
(If that functionality changes, it's likely that this documentation will not get updated.)

Reply:
Done!